ui/src/mock_data/logsData.ts

```diff
@@ -0,0 +1,134 @@
+const rawText = `Timestamp,Status,Transformer,Message
```

Comment:
For now this is fine. But keep in mind we will have actual RunResults being saved, so most likely we won't be getting a dump of results as a single string.

Reply:
Yea, I changed them into an object and the data is a JS object. I could have added json directly also.